R/latent_individual.R

```diff
@@ -205,12 +205,21 @@ epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
 #' @inheritParams epidist_prior
 #' @method epidist_prior epidist_latent_individual
 #' @family latent_individual
+#' @importFrom cli cli_inform
 #' @export
 epidist_prior.epidist_latent_individual <- function(data, ...) {
   validate_latent_individual(data)
 
   prior1 <- brms::prior("normal(2, 0.5)", class = "Intercept")
   prior2 <- brms::prior("normal(0, 0.5)", class = "Intercept", dpar = "sigma")
+
+  cli::cli_inform(c(
+    "i" = "The following priors have been set:",
```

Comment:
I think you need to be what the link function is for each or this is confusing.

Reply:
I have written something corresponding more directly to what the `brms` prior is. I think that we still have some confusion about the priors and families which we will need to handle (e.g. #131, #171). So I'd be in favour of this PR just saying what there is at the moment, and it's something we can come back to once we sort out how we are handling families.